RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Git Town Release Notes
 
+## Unreleased
+
+* `git sync --all`: push tags
```

Comment:
Yeah. Read more about changelogs when I added it to other repos and think this is nice. Also makes the actual releases easier.